controller/RestEligibility.php

```diff
@@ -357,12 +399,44 @@ private function getTakersFromRequest()
         } catch (\common_exception_MissingParameter $e) {
             return $result;
         }
+
         if (is_array($ids)) {
+            $result = $this->getTestTakerResources($ids, $result);
+        } else {
+            throw new \common_exception_RestApi(__('`%s` parameter must be an array', self::PARAMETER_TEST_TAKER_IDS));
+        }
+        return $result;
+    }
+
+    /**
+     * Get value for proctored eligibility from request.
+     *
+     * @return bool|null
+     */
+    private function getProctoredFromRequest()
+    {
+        $proctored = null;
+        if ($this->getRequest()->hasParameter(self::PARAMETER_ELIGIBILITY_PROCTORED)) {
+            $proctored = filter_var($this->getRequest()->getParameter(self::PARAMETER_ELIGIBILITY_PROCTORED), FILTER_VALIDATE_BOOLEAN);
```

Comment:
To keep consistency you could fetch the parameter by using `getParameterFromRequest()`.

Reply:
Done